src/client/app/containers/MultiCompareChartContainer.ts

```diff
@@ -181,4 +182,19 @@ function sortIDs(ids: CompareEntity[], sortingOrder: SortingOrder): CompareEntit
 	return ids;
 }
 
+function areaNormalizationValid(state: State, id: number, isGroup: boolean): boolean {
+	if(!state.graph.areaNormalization) {
+		return true;
+	}
+	if(isGroup) {
+		if(state.groups.byGroupID[id].area !== 0 && state.groups.byGroupID[id].areaUnit !== AreaUnitType.none) {
+			return true;
+		}
+	}
+	if(state.meters.byMeterID[id].area !== 0 || state.meters.byMeterID[id].areaUnit !== AreaUnitType.none) {
```

Comment:
Should this be the same as groups above where need an area AND a unit?

Reply:
Yep, should be an and statement.

I also realized that I've been using `area !== 0` in some cases, and `area > 0` in others. Do you have a preference which one should be used? They both have the same effect because area cannot be negative.

Reply:
I don't have a reason to prefer one over the other. Do whatever you want/is easier. Thanks for noticing this slight variation in the checks.